Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -21,7 +21,7 @@
       Supports the unit test frameworks MSTest2, NUnit3, XUnit2, MSpec, and NSpec3.
     </PackageDescription>
     <PackageProjectUrl>https://www.fluentassertions.com</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/fluentassertions/fluentassertions.git</RepositoryUrl>
+    <RepositoryUrl>https://github.com/fluentassertions/fluentassertions</RepositoryUrl>
```

Comment:
hmm...
https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets has the `.git` extension.

Reply:
What are you trying to say?

Reply:
That I'm not sure why we would want to remove the `.git` suffix from the url, as the documentation for that attribute uses it.

Reply:
Ah, I get it. I think the `.git` extension is deprecated. If you go to a URL with that postfix, you'll get redirected to the one without it.

Reply:
To actually answer your question, I didn't knew if it made any difference.
But digging into to it, 
[MSBuild](https://github.com/dotnet/msbuild/blob/47be9829672e2c207626419971aa1cba8f445409/Directory.Build.props#L23) doesn't use the `.git` extension, so I'm fine removing it.